test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    /**
+     * Test isAnyNull method using 2 objects
+     */
+
+    @Test
+    public void isAnyNull_noNull() {
+	Object obj1 = new Object();
+	Object obj2 = new Object();
+	final boolean result = Utils.isAnyNull(obj1, obj2);
+	assertFalse(result);
+    }
+
+    @Test
+    public void isAnyNull_hasNull() {
+	Object obj1 = new Object();
+	Object obj2 = null;
+	final boolean result = Utils.isAnyNull(obj1, obj2);
+	assertTrue(result);
+    }
+
+    /**
+     * Test elementsAreUnique method
+     */
+
+    @Test
+    public void elementsAreUnique_isUnique() {
+	ArrayList<Object> list = new ArrayList<Object>();
+	Object obj1 = new Object();
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.